spec/requests/can_respond_with_collection_of_latest_articles_spec.rb

```diff
@@ -0,0 +1,73 @@
+RSpec.describe 'GET /api/articles', type: :request do
+  let!(:test_experience) { create(:article, article_type: 'experience') }
+  let!(:test_article1) { create(:article, created_at: Time.current - 5.days) }
+  let!(:test_article2) { create(:article, title: 'Test Article 2', created_at: Time.current - 2.days) }
+  let!(:test_article3) { create(:article, created_at: Time.current - 3.days) }
+  let!(:test_article4) { create(:article, created_at: Time.current - 4.days) }
+  let!(:test_article5) { create(:article, title: 'Test Article 5', created_at: Time.current - 1.days) }
+  describe 'successfully' do
+    before do
+      get '/api/articles',
+          params: {
+            article_type: 'story'
+          }
+    end
+
+    it 'responds with a 200 status' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'filters out experience article, thus responding with a list of 5 articles' do
+      expect(response_json['articles'].length).to eq 5
+    end
+
+    it 'responds with newest article first in the list' do
+      expect(response_json['articles'].first['title']).to eq 'Test Article 5'
+      expect(response_json['articles'].second['title']).to eq 'Test Article 2'
```

Comment:
one expect statement per block please

Reply:
Even though it's two sides of the same coin?

Reply:
Yes, then you make another it block that tests for the other side of the coin